core/primitives-core/src/profile.rs

```diff
@@ -120,23 +84,13 @@ impl ProfileData {
         self.all_gas() - self.host_gas() - self.action_gas()
     }
 
-    pub fn unaccounted_gas(&self) -> u64 {
```

Comment:
Is this removed since it is calling all public methods?

Reply:
No, this is just a misleading method which always returns zero due to the way `wasm_gas` is defined.

Reply:
Ah I see.